scraping/spiders/poland_rjps.py

```diff
@@ -0,0 +1,119 @@
+from collections import defaultdict
+from typing import List, Dict
+
+import scrapy
+import json
+
+from joblib import Memory
+from config.settings import Settings
+
+memory = Memory(location='cache/poland_rjps')
+settings = Settings()
+DATA_PATH = f'{settings.spider_data_path}/rips'
+COUNTRY_NAME = 'Poland'
+DEFAULT_CATEGORY = "Any Help"
+
+CATEGORY_MAPPING = {
+    'Medical': ['seniors.json', 'health.json'],
```

Comment:
I think these JSONs should be scraped dynamically too instead of coming from static files in `DATA_PATH`

Reply:
https://github.com/littlepea/safeway-data/pull/18/commits/0a48f991ef028a8da748bd9c6c31cd4003080235